scripts/install/Install.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014 (original work) Open Assessment Technologies SA;
+ *
+ *
+ */
+
+namespace oat\taoDeliverySchedule\scripts\update;
+
+use oat\tao\model\ClientLibRegistry;
+
+/**
+ * 
+ * @author Aleh Hutnikau <hutnikau@1pt.com>
+ */
+$clientLibRegistry = ClientLibRegistry::getRegistry();
+$clientLibRegistry->register('moment', '../../../taoDeliverySchedule/views/js/lib/moment-with-locales.min');
```

Comment:
Do you absolutely need the alias moment for this to work ? If so please add a comment here so another won't be tempted to remove it.
However if it is not required, you can simply load it by requiring it as usual.
`define(['taoDeliverySchedule/lib/moment-with-locales.min'], function(moment){ ... });`